pkg/render/common/data/security.go

```diff
@@ -0,0 +1,77 @@
+package data
```

Comment:
I think we should switch these into their own package of `render/common/security`. I'd even consider splitting the functions into their own packages of `render/common/podsecuritypolicy` and `render/common/securitycontext`.

Reply:
Will do